src/content/docs/guides/hacktoberfest.md

```diff
@@ -0,0 +1,63 @@
+:::
+
+## Join the fest
+
+To take part in Hacktoberfest, here's all you need to do:
+
+1. Register for the event on [Hacktoberfest's official website](https://hacktoberfest.com/).
+2. Make any [valid contributions](#contributions) following [Hacktoberfest's rules](https://hacktoberfest.com/participation/#pr-mr-details) during October.
+3. Let us know you're participating by requesting your PRs to be labeled as `hacktoberfest-accepted`.
+
+## Primer on open-source
+
+Especially for first-time contributors, open-source can be quite frightening -- but don't worry, we're here to help you succeed! Here are a few reliable resources you can read to get up to speed on the matter:
+
+- [What is open-source](https://www.digitalocean.com/community/tutorials/what-is-open-source)
+- [How to contribute to open-source](https://opensource.guide/how-to-contribute/)
+- [How to use Git](https://www.digitalocean.com/community/cheatsheets/how-to-use-git-a-reference-guide)
+- [Getting started with GitHub Desktop](https://docs.github.com/en/desktop/overview/getting-started-with-github-desktop)
+
+## Contributions
+
+Any contribution to [participating repositories](#participating-repositories) will be considered and count towards Hacktoberfest. Our only exceptions are contributions that are either spam attempts or that have no quality and effort.
+
+:::caution
+AI contributions will not be accepted. Hacktoberfest exists to incentivize human contribution to open-source projects, and therefore, we cannot accept contributions that have been made by automated tools.
+
+This does not apply to AI assistance tools, such as GitHub Copilot or Cursor, that do not take over the core work of contributing from you.
+:::
+
+## Participating repositories
+
+### Astro Docs
+
+The [`withastro/docs` repository](https://github.com/withastro/docs) contains the complete source code and content for the [Astro documentation](https://docs.astro.build/) website in all the available languages.
+
+Here are some of the main ways you can contribute to it:
+
+- Reading the current or [5.0 beta docs](https://5-0-0-beta.docs.astro.build/) and fixing any typos, grammar issues, or missing information.
+- Helping [translate the documentation](http://contribute.docs.astro.build/guides/i18n/) in your language.
+- Tackling [open "good first issues"](https://github.com/withastro/docs/issues?q=sort%3Aupdated-desc+is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) in the repository.
+
+:::note[TODO]
+Adding information on the other participating repositories, I need to confirm.
+:::
+
+## Rewards
+
+All participating contributors will get thematic [Hacktoberfest Astro Badges](https://astro.badg.es/achievements/hacktoberfest-merges/bronze/) that "level up" as you contribute. You can showcase your Astro badges in a daily updated profile card, like this one: 
+
+[![@jsparkdev Astro contributions](https://astro.badg.es/v2/contributor/jsparkdev.svg)](https://astro.badg.es/contributor/jsparkdev/)
+
+
```

Comment:
[Enter your GitHub username and find your own contributor badge!](https://astro.badg.es/contributors/)